### PR TITLE
docs: shared responsibility model alignment documenation

### PR DESCRIPTION
**Pull request by felixzieger**

### CODE REVIEW

File: kit/foundation/docs/template/docs/foundation/README.md [hunk 1]
```diff
@@ -1,3 +1,88 @@
...
+
+## Your way to the cloud
+
+Cloud adoption has two phases:
+- The planning phase often begins with our consulting hours: Here you can inform yourself and ask your questions. In this phase you will choose a cloud provider and landing zone. Please note that not every service is available from every provider and landing zone. We recommend to create a solution design at this stage. Our cloud solution architects will support you in this.
+- The implementation phase begins when you have access to your cloud tenant. Now it's time for the big move - migrate your application and if necessary your data!
+
+## Cloud Providers
+
+> 🚧 Adapt the list of cloud providers you want to offer.
```
[the remark this answers is not in the document]
Comment (JohannesRudolph) [reply kit/foundation/docs/template/docs/foundation/README.md:11]: and: if you have a company wide cloud strategy, here's also a good place to link to it

File: kit/foundation/docs/template/docs/foundation/README.md [hunk 2]
```diff
@@ -1,3 +1,88 @@
...
+The following responsiblities are with the cloud provider for all landing zones:
+- Physical hosts
+- Physical network
+- Physical datacenter
+
+The following responsibilities are with the cloud foundation team for all landing zones:
+- Identities, SSO on cloud platform
+- Audit Logs for calls against cloud platform APIs
+
+The following responsibilities are with the applicaiton team for all landing zones:
```
Comment (JohannesRudolph) [inline kit/foundation/docs/template/docs/foundation/README.md:36]: typo (application)

File: kit/foundation/docs/template/docs/foundation/README.md [hunk 3]
```diff
@@ -1,3 +1,88 @@
...
+We offer [landing zones](concepts.md#landing-zone) for each of those [platforms](/platforms/).
+
+## Shared Responsibility Model
+
+For our cloud offering, we rely on sharing responsibilities across three parties: cloud provider, cloud foundation team, and you (the application team).
+It’s critical to understand which security tasks are handled by whom.
+
+## Common responsibilities across all landing zones
+
+The following responsiblities are with the cloud provider for all landing zones:
```
Comment (JohannesRudolph) [inline kit/foundation/docs/template/docs/foundation/README.md:27]: maybe use a markdown table here

File: kit/foundation/docs/template/docs/foundation/README.md [hunk 4]
```diff
@@ -1,3 +1,88 @@
...
+The following responsibilities are with the cloud foundation team for all landing zones:
+- Identities, SSO on cloud platform
+- Audit Logs for calls against cloud platform APIs
+
+The following responsibilities are with the applicaiton team for all landing zones:
+- Permissions on the cloud platform
+
+## Landing Zones and Responsibilities
+
+> 🚧 Adapt the list of landing zones you want to offer.
```
Comment (JohannesRudolph) [inline kit/foundation/docs/template/docs/foundation/README.md:41]: Is there a resource we can link to for more details? The ultimate landing zone guide comes to mind, LZ101 or the CFMM.

Comment (JohannesRudolph) [reply kit/foundation/docs/template/docs/foundation/README.md:41]: Add sentence: "In this example here we've chosen to divide landing zones by use case (Lift and Shift and cloud-native workloads) as well as network connectivity to on-prem.

Comment (JohannesRudolph) [reply kit/foundation/docs/template/docs/foundation/README.md:41]: can also have an additional addendum: this division is a very common starting point for cloud foundation teams

File: kit/foundation/docs/template/docs/foundation/README.md [hunk 5]
```diff
@@ -1,3 +1,88 @@
...
+Because this landing zone does not connect to on-prem, this landing zone is a popular choice for testing out the capabilities of a cloud provider.
+
+The following responsibilites are with the application team:
+- Development and operations of your application
+- Identities and Accounts within your application
+- Information and data within your application
+- Securing all endpoints of your application
+- Network controls
+- Operating system
+- Costs
```
Comment (JohannesRudolph) [inline kit/foundation/docs/template/docs/foundation/README.md:87]: costs is always with application team, should move it to the top (pre-landing zones)

File: kit/foundation/docs/template/docs/foundation/concepts.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Concepts
+
+## Landing Zone
```
Comment (JohannesRudolph) [inline kit/foundation/docs/template/docs/foundation/concepts.md:3]: we can link to CFMM here, though I'm not sure there's a single page that explains LZ, e.g. in the CFMM Knowledge Hub.

That probably needs a bit of a makeover as well https://cloudfoundation.org/understanding-cloud-foundation/#why-build-a-cloud-foundation

Comment (JohannesRudolph) [reply kit/foundation/docs/template/docs/foundation/concepts.md:3]: (though that's out of scope for collie-hub here today)

**Review by JohannesRudolph — APPROVED** [review]

**Comment by felixzieger** [comment]
Thanks for the tips, @JohannesRudolph. Adapted the docs and will merge when the checks are through

**Pull request merged**